src-frontend/app/components/procedure-card.js

```diff
@@ -1,5 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-    tagName: 'li'
+    tagName: 'li',
+
+    actions: {
+        deleteProcedure: function(procedure) {
+            procedure.deleteRecord();
+            procedure.save();
```

Comment:
I think we can just call `procedure.destroyRecord()` instead of calling `deleteRecord()` and `save()`

Ref: http://guides.emberjs.com/v1.10.0/models/creating-and-deleting-records/